app/src/main/java/com/keylesspalace/tusky/util/AdaptiveTabLayout.kt

```diff
@@ -0,0 +1,46 @@
+package com.keylesspalace.tusky.util
```

Comment:
should be in the `com.keylesspalace.tusky.view` package

Reply:
ok